dts/arm/nxp/nxp_kl25z.dtsi

```diff
@@ -39,6 +39,7 @@
 			compatible = "nxp,kinetis-ftfa";
 			reg = <0x40020000 0x14>;
 			interrupts = <5 0>;
+			status = "disabled";
```

Comment:
Why is this disabled?  (at least add comment in commit message)

Reply:
Will update commit message, but the flash controller is disabled on this platform because it's not currently tested to work on the kl25z soc.